ui/app/templates/variables/variable/index.hbs

```diff
@@ -36,7 +36,7 @@
   </div>
   <div>
     {{#unless this.isDeleting}}
-      {{#if (can "write variable" path=this.model.absolutePath)}}
+      {{#if (can "write variable" path=this.model.path)}}
```

Comment:
Out of curiosity, why `absolutePath` to `path` ? absolutePath includes everything from root on forward, no?

Reply:
`absolutePath` does not exist on the model. The `variables.variable.index` route does not return multiple models by creating the `PathTree` presumably because `variables.variable.index` is displaying information about a single variable and not listing folder paths (where I believe that `absolutePath` logic is required).